package.json

```diff
@@ -29,7 +29,7 @@
     "ophan-tracker-js": "1.3.13",
     "preact": "^8.2.9",
     "preact-compat": "^3.18.4",
-    "prebid.js": "https://github.com/guardian/Prebid.js.git#71d18d2",
+    "prebid.js": "https://github.com/guardian/Prebid.js.git",
```

Comment:
This will implicitly change when master updates, which is why we use direct commit references. So you might want to use the commit new/old ID.

Reply:
:+1: added it